stl/inc/ranges

```diff
@@ -1379,11 +1382,9 @@ namespace ranges {
         return basic_istream_view<_Ty, _Elem, _Traits>{_Stream};
     }
 
-    // clang-format off

```

Comment:
I am a bit unsure about that one. Usually clang-format would bail here

Reply:
I don't know why clang-format wants to actually do the right thing here, but it does. This region is definitely "clang-format on" (from line 1312). I discovered this when I damaged the comments while copying `ref_view` to `owning_view`.